knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -271,7 +277,9 @@ func (r *ReconcileKnativeServing) ensureCustomCertsConfigMap(instance *operatorv
 	if err != nil {
 		return fmt.Errorf("error reconciling serviceCACM: %w", err)
 	}
-	trustedCACM, err := r.reconcileConfigMap(instance, certs.Name+"-trusted-ca", nil, map[string]string{trustedCAKey: "true"}, nil)
+	trustedCACM, err := r.reconcileConfigMap(instance, certs.Name+"-trusted-ca",
```

Comment:
I guess that is the only cm we should take care right?

Reply:
I think so yes, as this is not a global thing.